src/schemathesis/specs/openapi/negative/utils.py

```diff
@@ -0,0 +1,10 @@
+from typing import List
+
+from .types import Schema
+
+
+def get_type(schema: Schema) -> List[str]:
+    type_ = schema.get("type", ["null", "boolean", "integer", "number", "string", "array", "object"])
```

Comment:
Just curious, why `_` at the end of the variable name?

Reply:
Because "type" is the name of a Python built-in!